<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>476</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>175</x>
      <y>470</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QDial" name="dial_set_point">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>310</y>
      <width>161</width>
      <height>131</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>381</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Fuzzy Temperature Controller</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_title">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>100</y>
      <width>400</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_set_point">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>200</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_set_point">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>381</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set Point</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>476</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
